app/assets/javascripts/autoform_form.js

```diff
@@ -29,33 +44,19 @@ $(document).ready(function() {
             return true;
         }
     });
-
-    var autoformError = $('#autoform-error').get(0);
-    addCloseButtonListener(autoformError);
```

Comment:
chcel som označiť tento riadok, keď sa nenájde autoformError z predchádzajúceho riadku nič sa nestane, no keď sa potom posunie do funkcie `addCloseButtonListener(autoformError);`, tak to zlyhá na `    var closeButton = node.querySelector('.alert .close');
`, preto sa toto teraz udeje iba keď sa nájde `autoformError`